authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc

```diff
@@ -81,6 +81,13 @@ data:
 <3> The resource URL of the service account that can be impersonated with these credentials.
 <4> The path to the service account token inside the pod. By convention, this is `/var/run/secrets/openshift/serviceaccount/token` for {product-title} components.
 
+[IMPORTANT]
+====
+When using manual mode with GCP Workload Identity, the {product-title} image registry experiences degraded performance. In {product-title} 4.10.8, support for using GCP Workload Identity for the image registry is removed from {product-title} 4.10.
```

Comment:
I think we can get rid of the first sentence here. That's what we're fixing in 4.10.8, the slow performance, and that fix leads to the rest of the stuff we're talking about. Maybe consider bringing the two remaining sentences in this admonition together / cleaning them up?

Reply:
And actually, I don't beleive that the performance regression needed the cluster to use Workload Identity, it's just the code changes we made for WI led to the regression which is present in all situations.

Reply:
Ah, interesting detail :) Ok, I will revise as you suggest in all occurrences of this text :+1: